swagger_server/controllers_local/collectioninstrument.py

```diff
@@ -0,0 +1,300 @@
+##############################################################################
+#                                                                            #
+#   Collection Instruments Upload                                            #
+#   Date:    11 May 2017                                                     #
+#   Author:  Gareth Bult                                                     #
+#   License: MIT                                                             #
+#   Copyright (c) 2017 Crown Copyright (Office for National Statistics)      #
+#                                                                            #
+##############################################################################
+from ..configuration import ons_env
+from ..models_local.instrument import InstrumentModel
+from ..models_local.exercise import ExerciseModel
+from ..models_local.business import BusinessModel
+from ..models_local.survey import SurveyModel
+from ..models_local.classification import ClassificationModel, classifications
+from traceback import print_exc
+from sys import stdout
+from json import loads
+from uuid import UUID
+
+DEFAULT_SURVEY = "3decb89c-c5f5-41b8-9e74-5033395d247e"
```

Comment:
Yes, this is the UUID of "BRES". This is needed for current uploads that do not provide a uid for the survey in use, yet we need to associate uploads with a given survey (uid). This will change once we know more about how we will interact with other uploads. It might be good to have an application specific ini file to put such constants in, but that infrastructure doesn't exist yet.